jest-environment.js

```diff
@@ -1,17 +1,26 @@
+/**
+ * @license BSD-3-Clause
+ *
+ * Copyright (c) 2019 Project Jupyter Contributors.
+ * Distributed under the terms of the 3-Clause BSD License.
+ */
+
 // Based on from https://yarnpkg.com/en/package/@rws-air/jestscreenshot
 
-const PuppeteerEnvironment = require("jest-environment-puppeteer");
-const JestScreenshot = require("@rws-air/jestscreenshot");
-require("jest-circus");
```

Comment:
It's interesting you didn't need this! Not sure why we did, they just had it here.

Reply:
This was present in one of the dep documentation examples. I am not sure why it was there. If it was needed, I would disappointed, as it means importing the package has (gasp) side-effects :|.